### PR TITLE
Storage reporter upgrade

### PR DESCRIPTION
**Pull request by janezpodhostnik**
Closes: https://github.com/dapperlabs/flow-go/issues/5133

The storage reporter will create a file with accounts statistics:

- address
- storage used in bytes
- flow balance
- has default vault
- has dapperUtilityCoinReceiver (dapper accounts)

This data will be dumped to a file.

The `StorageFeesMigration` was removed from the migrations list because it was already applied to both mainnet and testnet.

### CODE REVIEW

File: cmd/util/ledger/migrations/storage_reporter.go [hunk 1]
```diff
@@ -35,76 +65,96 @@ func (r StorageReporter) Report(payload []ledger.Payload) error {
...
-		}
+	return nil
+}
 
-		graph[i] = int(math.Ceil((v / maxBin) * float64(binHeight)))
+func (r StorageReporter) isDapper(address flow.Address, st *state.State) (bool, error) {
+	id := flow.RegisterID{
+		Owner:      string(address.Bytes()),
+		Controller: "",
+		Key:        fmt.Sprintf("%s\x1F%s", "public", "dapperUtilityCoinReceiver"),
```
Comment (turbolent) [inline cmd/util/ledger/migrations/storage_reporter.go:99]: Would be nice if we could use a function here. Maybe refactor this and the other instances below into a function and link to the Cadence function that does this.

File: cmd/util/ledger/migrations/storage_reporter.go [hunk 2]
```diff
@@ -35,76 +65,96 @@ func (r StorageReporter) Report(payload []ledger.Payload) error {
...
+	}
 
-	return nil
+	balanceId := flow.RegisterID{
+		Owner:      string(address.Bytes()),
+		Controller: "",
+		Key:        fmt.Sprintf("%s\x1F%s", "public", "flowTokenBalance"),
+	}
+
+	balanceResource, err := st.Get(balanceId.Owner, balanceId.Controller, balanceId.Key)
```
Comment (turbolent) [inline cmd/util/ledger/migrations/storage_reporter.go:122]: Maybe rename to `balanceCapability`

**Review by turbolent — APPROVED** [review]
Nice!

File: cmd/util/ledger/migrations/storage_reporter.go (hunk 2 above)
Comment (janezpodhostnik) [reply cmd/util/ledger/migrations/storage_reporter.go]: Updated

**Review by ramtinms — APPROVED** [review]

**Pull request merged**